test/libsolidity/smtCheckerTests/special/msg_vars_chc_internal.sol

```diff
@@ -21,8 +21,11 @@ contract C {
 		assert(sender == msg.sender); // should hold with CHC
 		assert(sig == msg.sig); // should hold with CHC
 		assert(value == msg.value); // should hold with CHC
+
+		assert(msg.value == 10); // should fail
 	}
 }
 // ====
 // SMTEngine: chc
 // ----
+// Warning 6328: (621-644): CHC: Assertion violation happens here.\nCounterexample:\ndata = [38, 18, 31, 240], sender = 0, sig = 638722032, value = 0\n\nTransaction trace:\nC.constructor()\nState: data = [], sender = 0, sig = 0, value = 0\nC.f(){ msg.data: [38, 18, 31, 240], msg.sender: 0, msg.sig: 638722032, msg.value: 0 }\n    C.g() -- internal call
```

Comment:
For data or sig, I think hex is a better choice. But fine either way.

Reply:
I think so too. Let me see if I can do this in an easy way, otherwise I'd vote to do it in a different PR.